### ISSUE TITLE
Project specific build commands

### ISSUE THREAD

**Issue by Selyss** [trimclain/builder.nvim#2]
I believe an easy way to specify a build command per project would be very useful and helpful for people with more complex build needs in certain projects. The process for adding project specific builds would need to be fairly intuitive for the end user, and not require a ton of mental overhead to use. The data could be stored in some kind of json file on the users computer, or something of the sort.

**Comment by trimclain** [comment]
Good Idea. I'm thinking adding the ability to read something like `.builder.json` from the root of a project.
But we should probaby think about this [problem](https://github.com/nodejs/tooling/issues/79) aswell :rofl:
